tests/test_parse.py

```diff
@@ -4,6 +4,7 @@
 from datetime import timezone
 from enum import Enum
 from enum import IntEnum
+from enum import StrEnum
```

Comment:
you will have to use local-scope import with some Python dirtiness to support this feature:
```python
  ..
  if ..:
    try:
      from enum import StrEnum
    except ImportError:
      # StrEnum not available
  ..
```
Does that help? @bakobako